tapiriik/services/Decathlon/decathlon.py

```diff
@@ -224,6 +226,55 @@ def RevokeAuthorization(self, serviceRecord):
         '''
         pass
 
+
+    class AuthenticationRefreshResponse:
+        HttpCode: int
+        CorrelationId: str
+        ExternalId: str
+        Body: dict
+
+        IsInError: bool=False
+        ErrorDescription: str=""
+        
+        def __init__(self, response: requests.Response, external_user_id: str):
+            self.HttpCode = response.status_code
+            self.ExternalId = external_user_id
+            self.CorrelationId = response.headers.get("X-Correlation-Id", "NULL")
+
+            try:
+                self.Body = response.json()
+            except JSONDecodeError:
+                self.Body = {}
+
+            if response.status_code != 200:
+                self.IsInError = True
+                self.ErrorDescription = self.Body.get("error_description", "")
+            
+
+        def is_blocking(self) -> bool:
+            if (self.HttpCode == 400
```

Comment:
think you can just do this : 
`return self.HttpCode == 400
                    or self.HttpCode == 401 
                    or self.HttpCode == 403`

Reply:
or maybe even 
`return self.HttoCode in [400, 401, 403]`
If it's correct python syntax :)